src/Appium.Net/Appium/AppiumCommand.cs

```diff
@@ -194,6 +194,9 @@ public class AppiumCommand
             #region Compare Images
             new AppiumCommand(CommandInfo.PostCommand, AppiumDriverCommand.CompareImages, "/session/{sessionId}/appium/compare_images" ),
             #endregion
+
+            new AppiumCommand(CommandInfo.GetCommand, DriverCommand.GetAvailableLogTypes, "session/{sessionId}/log/types"),
```

Comment:
I thought these endpoints were already present for selenium

Reply:
I think it is available - let me check

Reply:
I cannot remember off the top of my head why I added that, they definitely exist in selenium, but I think maybe the CommandInfo value differs to appium. I'll run the tests without the entries in a moment. Will confirm after. In other words, don't merge yet :)

Reply:
Thanks @akinsolb

Reply:
It needs to be added because it's not in the W3CWireProtocolCommandInfoRepository

Reply:
Yes, the log stuff are not in W3C.
https://github.com/appium/ruby_lib_core/blob/master/lib/appium_lib_core/common/command/w3c.rb is commands used in Appium, but selenium w3c module does not have. OSS module had them.